.storybook/webpack.config.js

```diff
@@ -73,5 +73,19 @@ module.exports = webpackSettings => {
     })
   );
 
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: require.resolve('awesome-typescript-loader'),
```

Comment:
When I was playing around with this PR, it seemed like it was possible to generate types just using `babel-loader` along `@babel/preset-typescript`.

I think we should try and stay within the Babel ecosystem as much as possible. Did you play around with it at all?